### ISSUE TITLE
Frontend module copies source to wrong directory

### ISSUE THREAD

**Issue by camullen** [stackery/wild-rydes-workshop#22]
In the 01-Frontend.md file, there is a suggestion that the user clone the repo and copy the frontend code to their src directory:
```
cp -R ../wild-rydes-workshop/src/site src/
```

However, when deploying the stack, using the default configuration (as suggested by the tutorial), the CodeBuild resource expects the source to be in src/site

Suggest changing to:

```
mkdir -p src/site
cp -R ../wild-rydes-workshop/src/site src/site
```

**Comment by txase** [comment]
Hi @camullen!

Thanks for the suggestion! This may have been overlooked when we updated from a previous version that didn't use our newer "Website Builder" resource :).

🙏

**Comment by txase** [comment]
Fixed in version ac5cd307d5b560ca0ed24fb55772b5a9f95de7d4 :).

Thanks for reporting!
